qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/diagnostics/QuteErrorCode.java

```diff
@@ -62,6 +62,11 @@ public enum QuteErrorCode implements IQuteErrorCode {
 	MethodIgnoredByTemplateData(
 			"The method `{0}` of `{1}` Java type cannot be used in native image mode because Java type is annotated with `@TemplateData(ignore = {2})`."), //
 
+	// Error code for expression part

```

Comment:
// Error code for expression part in #case  / #is section

Reply:
I think we should have an error code which says that it is for #case (or #is). I will rename: 

 * UnexpectedMemberType -> UnexpectedMemberTypeInCaseSection
 *  UnexpectedPartValue-> UnexpectedValueInCaseSection
 *  MismatchingSection-> InvalidParentInCaseSection